_includes/navbar.html

```diff
@@ -3,11 +3,18 @@
 
 <nav id="header">
   <span id="left-side">
-    <a href="{{site.baseurl}}/" class="navbar_tab">Home</a>
-    <a href="{{site.baseurl}}/main/" class="navbar_tab">ACLE</a>
-    <a href="{{site.baseurl}}/mve_intrinsics/" class="navbar_tab">MVE</a>
-    <a href="{{site.baseurl}}/morello/" class="navbar_tab">Morello</a>
-    <a href="{{site.baseurl}}/neon_intrinsics/" class="navbar_tab">Neon</a>
+      <a href="{{site.baseurl}}/"
+        {% if page.url != '/' %} class="navbar_tab"
+        {% else %} class="navbar_active_tab"
+        {% endif %}>
+        Home</a>
+    {% for nav in site.data.navigation %}
+      <a href="{{site.baseurl}}{{nav.url}}"
+        {% if page.url contains nav.url %} class="navbar_active_tab"
```

Comment:
Can't you do something like the following?:

```suggestion
        {% if page.url == {{site.baseurl}}{{nav.url}} %} class="navbar_active_tab"
```

This would check the whole string, meaning that you could just add the following data to the yml file and avoid the custom code you need for the "home" link:

```
- text: Home
  url: /
- text: ACLE
  url: /main/
...
```

Reply:
I actually think it might just be page.url == nav.url, but yes

Reply:
Actually, I remember why I did it using `contains` - because otherwise the tabs won't show as active if you're on, say /morello/morello.html instead of just /morello/

Reply:
Does it work now with this code?

```
    {% for nav in site.data.navigation %}
      <a href="{{site.baseurl}}{{nav.url}}"
        {% if page.url == nav.url %} class="navbar_active_tab"
        {% else %} class="navbar_tab"
        {% endif %}>
        {{ nav.text }}</a>
    {% endfor %}
```

Reply:
I mean, it's functional for the README.md files, but not any other files within the folder.

Reply:
So it works only for the following addresses?

* arm-software.github.io/acle/
* arm-software.github.io/acle/main/
* arm-software.github.io/acle/morello/
* arm-software.github.io/acle/neon_intrinsics/
* arm-software.github.io/acle/mve_intrinsics/

Reply:
Yes